include/iarc7_motion/LandPlanner.hpp

```diff
@@ -0,0 +1,100 @@
+////////////////////////////////////////////////////////////////////////////
+//
+// Land Controller
+//
+// Handles basic landing
+//
+////////////////////////////////////////////////////////////////////////////
+
+#ifndef LAND_CONTROLLER_H
+#define LAND_CONTROLLER_H
+
+#include <ros/ros.h>
+
+#include "actionlib/server/simple_action_server.h"
+
+#include "ros_utils/LinearMsgInterpolator.hpp"
+#include "ros_utils/SafeTransformWrapper.hpp"
+
+// ROS message headers
+#include "geometry_msgs/TwistStamped.h"
+
+#include "iarc7_motion/GroundInteractionAction.h"
+
+namespace Iarc7Motion
+{
+
+typedef actionlib::SimpleActionServer<iarc7_motion::GroundInteractionAction> Server;
+
+enum class LandState { BEGIN_DESCENT,
+                       DESCEND,
+                       BRACE_FOR_IMPACT,
+                       DONE };
+
+    class LandPlanner
```

Comment:
I meant the other stuff shouldn't be indented, that's what I've been doing and I thought that was our convention